app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :user, foreign_key: 'author_id', counter_cache: :posts_counter
```

Comment:
- [ Optional] I recommend you to use author instead of user.  we can optimize like four in this way. `belongs_to :author, Class_name: 'User'`